tests/dummy/app/components/clock-demo.js

```diff
@@ -1,6 +1,22 @@
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
 
-export default class StopwatchDemoComponent extends Component {
+export default class ClockDemoComponent extends Component {
     @service clock;
+
+    @computed('clock.minute')
+    get timeByTheMinute() {
+        return this.clock.time;
```

Comment:
Should be `this.clock.minute`? Also, the other two computed properties may need to be updated as well. Thanks so much for your PR!

Reply:
Hmm no, not really. This is trying to illustrate that the `timeByMinute` only updates the template with a new value every minute as a result of the computed macro.

Perhaps a better example would be if I did:

```javascript
    @computed('clock.minute')
    get timeByTheMinute() {
        return (new Date()).getTime();
    }
```

So that readers don't think there's a mistake or a different dependency.

Reply:
I've gone ahead and updated the PR again. 

And no problem, I'm just glad there's a more modern version of a package like this, and I'm keen to get it to be a drop in replacement for https://github.com/lozjackson/ember-clock

:), certainly helps our project out.